addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1672,6 +1631,24 @@ options.registry.anchorName = options.Class.extend({
         }
         this.$target.trigger('content_changed');
     },
+    /**
+     * Returns anchor text.
+     *
+     * @private
+     */
+     _getAnchorLink: function (previewMode, value, $opt) {
+         if (!this.$target[0].id) {
+             const $titles = this.$target.find('h1, h2, h3, h4, h5, h6');
+             const title = $titles.length > 0 ? $titles[0].innerText : this.data.snippetName;
+             const anchorName = encodeURIComponent(title);
+             let n = '';
+             while (document.getElementById(anchorName + n)) {
+                 n = (n || 1) + 1;
+             }
+             this._setAnchorName(anchorName + n);
+         }
+         return $('<a>', {href: '#' + this.$target[0].id})[0].outerHTML;
```

Comment:
Sorry to make you change this again, Fabien went to see the feature and he actually wants the URL to be copied not the DOM Link... :) cc @sbuhl

Reply:
No problem! I thought it was weird to copy a DOM link since you can't really use them anywhere in the editor, which is why I did it that way the first time.